apiary.apib

```diff
@@ -2810,6 +2827,21 @@ Delete a quotation.
 
 + Response 204
 
+### quotations.requestExtension [POST /quotations.requestExtension]
+
+Request an extension for a quotation. Only possible for expired quotations.
+
++ Request (application/json)
+
+    + Attributes (object)
+        + id: `d885e5d5-bacb-4607-bde9-abc4a04a901b` (string, required)
+        + send_to: `info@teamleader.eu` (string, required)
+        + send_copy_to: maria@teamleader.eu` (string, optional, nullable)
+        + subject: `I missed the deadline' quotation` (string, required)
+        + content: `Can you reopen this quotation, please?` (string, required)
```

Comment:
PP: The example text is confusing, this looks like a request an end customer would do to our customer. However end customers will not call the api.

Reply:
But this is exactly what this endpoint does. Our customer will never request an extension, it will update the expiration date in that case. This is for extensions request by the end customer.

Maybe this endpoint should be reconsidered.